infra/src/submit.ts

```diff
@@ -13,20 +13,18 @@ async function main(): Promise<void> {
   const environment = [
     { name: 'LINZ_CORRELATION_ID', value: correlationId },
     { name: 'LINZ_SSM_BUCKET_CONFIG_NAME', value: 'BucketConfig' },
-    { name: 'LDS_CACHE_BUCKET', value: 'linz-lds-cache' },
+    { name: 'LINZ_CACHE_BUCKET', value: 'linz-lds-cache' },
```

Comment:
Did I miss something and `LDS_CACHE` is now named `LINZ_CACHE`?

Reply:
Oops yes that is from another PR, I've reverted as that PR is not yet merged.

Reply:
Ahh I renamed it so everything started with `LINZ_` no real reason just so it looks nice :D